ceph.spec.in

```diff
@@ -730,6 +730,15 @@ Requires:	libcephfs2 = %{_epoch_prefix}%{version}-%{release}
 %description -n cephfs-mirror
 Daemon for mirroring CephFS snapshots between Ceph clusters.
 
+%package -n ceph-exporter
```

Comment:
```suggestion
%package exporter
```

Reply:
`Exporter` seems a bit too generic given the functionality of this daemon in my opinion.